src/mpid/ch4/src/ch4r_win.h

```diff
@@ -798,6 +928,32 @@ static inline int MPIDI_CH4R_mpi_win_get_info(MPIR_Win * win, MPIR_Info ** info_
     if (MPI_SUCCESS != mpi_errno)
         MPIR_ERR_POP(mpi_errno);
 
+    {
```

Comment:
a format comment, why we need `{}` this code block till line 938 here? there is not `#define` macro inside like line 863.

Reply:
I want to keep char buf[128]; near the corresponding info handling code, similar to other info handling code. If I do not add {}, buf variable must be moved to beginning of the function, and each block cannot declare buf with different size. Then I have to rename all buf variables. It might be better to just add {} to avoid too much code change

Reply:
In that case maybe add a code comment? That way, someone looking at this code in the future will understand the motive and not get confused.

Reply:
Added.